rllib/examples/envs/env_rendering_and_recording.py

```diff
@@ -19,7 +19,9 @@
 --wandb-run-name=[optional: WandB run name within --wandb-project]`
 
 In order to see the actual videos, you need to have a WandB account and provide your
-API key and a project name on the command line (see above).
+API key and a project name on the command line (see above). To log the videos in WandB
```

Comment:
We should also add this to the requirements.txt

Here:
`python/requirements/ml/rllib-test-requirements.txt`

Just `moviepy`, wandb is already a ML core dependency.